### DISCUSSION TITLE
Is there considered separating the front-end UI from the back-end services?

### DISCUSSION THREAD

**Discussion by newyearsoft** [serenity-is/Serenity#5942] [Serenity Ideas]
It's a great template, and I love it. Is there considered separating the front-end UI from the back-end services?  For the trend, for the future!

**Comment by VictorTomaili** [comment]
Serenity has integrated features from back-end to front-end. If you want to use a separated front-end there is no restriction about that (no need to plan anything for the future). You can call services from anywhere. Currently, we don't have any samples for the separated front-end. But the community may have.
